samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,202 @@
+        AI_Model  - Generated while creating Azure Open AI service 
+        SubscriptionURL - "https://graph.microsoft.com/v1.0/subscriptions"
+        SystemPrompt="Generate a filtered list of action items from meeting transcriptions by user in bullet point user wise categorized with proper format like:  <b> Attendee:</b> 
+        Action Items in bullet points"
+        LocalTimeZone=Asia/Kolkata
+        APPINSIGHTS_INSTRUMENTATIONKEY- Provide the application insights `Instrumentation Key` created in previous steps (Required to log the telemetry data).
+        APPINSIGHTS_CONNECTIONSTRING  - Provide the application insights `Connection String` created in previous steps in single quote (Required to log the telemetry data).
+    ``` 
+1) Create a policy for a demo tenant user for creating the online meeting on behalf of that user using the following PowerShell script (Images for reference only)
+ 
+      PowerShell script
+
+    ```powershell
+    # Import-Module MicrosoftTeams
+    # Call Connect-MicrosoftTeams using no parameters to open a window allowing for MFA accounts to authenticate
+    Connect-MicrosoftTeams
+    New-CsApplicationAccessPolicy -Identity “<<policy-identity/policy-name>>” -AppIds "<<microsoft-app-id>>" -Description "<<policy-description>>"
+    Grant-CsApplicationAccessPolicy -PolicyName “<<policy-identity/policy-name>>” -Identity "<<object-id-of-the-user-to-whom-the-policy-needs-to-be-granted>>"
+    
+    OR
+    # For global access
+    # Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Global
+    ```
+    Example:
+
+    ```powershell
+      # Import-Module MicrosoftTeams
+      Connect-MicrosoftTeams
+
+      New-CsApplicationAccessPolicy -Identity Meeting-policy-dev -AppIds "xxxxxx0f-xxe2-4exx-9exx-2exxxx76bxxc" -Description "Online meeting policy - contoso town"
+      
+      Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Identity "7xxxx076x-xxxx-4xfx-x6x3-xa1xxx28xxxc"
+      OR
+      # For global access
+      # Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Global
+    ```
+    
+     ![PolicySetup](Images/Policy.png)
+     
+1) In Azure [App Registration](https://ms.portal.azure.com/) Under left menu, navigate to **API Permissions**, and make sure to add the following permissions of Microsoft Graph API > Application permissions:
+
+    - `Calendars.Read`
+    - `Calendars.ReadBasic.All`
+    - `Calendars.ReadWrite`
+    - `EventListener.Read.All`
+    - `EventListener.ReadWrite.All`
+    - `OnlineMeetingArtifact.Read.All`
+    - `OnlineMeetingRecording.Read.All`
+    - `OnlineMeetings.Read.All`
+    - `OnlineMeetings.ReadWrite.All`
+    - `OnlineMeetingTranscript.Read.All`
+    - `User.Read.All`
+    - `User.ReadBasic.All`
+    - `User.ReadWrite.All`
+
+        ![Application Permission](Images/ApplicationPermission.png)
+
+1) Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
```

Comment:
This line  looks similar to next ## Run the app (Using Teams Toolkit for Visual Studio Code)

Reply:
removed